README.md

```diff
@@ -124,6 +124,57 @@ Using JSON payload for constructing a message is also available:
     SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
 ```
 
+### Update the message
+
+Using `update-ts: ${{ steps.slack.outputs.ts }}` will update the message that was posted. 
+
+```yaml
+- id: slack
+  uses: slackapi/slack-github-action@v1.18.0
+  with:
+    channel-id: "#channel"
+    payload: |
+      {
+        "attachments": [
```

Comment:
Having the top-level text too is recommended for better user experience (readable mobile notification, more compatibility with the built-in accessibility features)
```suggestion
        "text": "Deployment started (In Progress)",
        "attachments": [
```